osu.Game/Overlays/BeatmapListing/SearchExtra.cs

```diff
@@ -1,38 +1,20 @@
 ﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
-using System;
 using System.ComponentModel;
 using osu.Framework.Localisation;
 using osu.Game.Resources.Localisation.Web;
 
 namespace osu.Game.Overlays.BeatmapListing
 {
-    [LocalisableEnum(typeof(SearchExtraEnumLocalisationMapper))]
     public enum SearchExtra
     {
+        [LocalisableDescription(typeof(BeatmapsStrings), nameof(BeatmapsStrings.ExtraVideo))]
         [Description("Has Video")]
         Video,
 
+        [LocalisableDescription(typeof(BeatmapsStrings), nameof(BeatmapsStrings.ExtraStoryboard))]
         [Description("Has Storyboard")]
```

Comment:
Is there any point in leaving the base `DescriptionAttribute`s here?

Reply:
There might potentially be a case where `GetDescription` is used rather than `GetLocalisableDescription`, and is out-of-scope from attempting that given that the description attribute was still here before this PR.